Clean docker and make build fails on Windows WSL

#### Version of the game
 - Portal64 v11
 - Windows 10 | WSL 2 (Ubuntu)

#### What is the bug?
I tried to build from scratch with the docker image and the manual make command. Both fail with the same error message:
`mips-n64-gcc -c -mabi=32 -ffreestanding -mfix4300 -G 0     -DDEBUG -g -Isrc/ -I/usr/include/n64/nustd -Werror -Wall -DF3DEX_GBI_2 -DSCENE_SCALE=128 -I/usr/include/n64/PR    -I/usr/include/n64 -O0    -MM src/util/dynamic_asset_loader.c -MF "build/src/util/dynamic_asset_loader.d" -MT"build/src/util/dynamic_asset_loader.o"`

`src/util/dynamic_asset_loader.c:5:10: fatal error: ../build/assets/models/dynamic_model_list.h: No such file or directory
    5 | #include "../build/assets/models/dynamic_model_list.h"`

#### Steps to reproduce the behavior:
1. Follow the build guide in the readme until the make commands
2. Notice that the build fails

#### Screenshots
![grafik](https://github.com/lambertjamesd/portal64/assets/3209654/e82b98e2-3a97-4d30-8a2a-8435f72dd9ed)
![grafik](https://github.com/lambertjamesd/portal64/assets/3209654/8953e079-11be-4b18-8925-1ec59e091520)
![grafik](https://github.com/lambertjamesd/portal64/assets/3209654/a11fc5eb-f5ac-4ad5-a14c-93a999e19a28)

Cheers, I get a little further in the make process, but am now stuck at:
`mips-n64-gcc -c -mabi=32 -ffreestanding -mfix4300 -G 0     -DDEBUG -g -Isrc/ -I/usr/include/n64/nustd -Werror -Wall -DF3DEX_GBI_2 -DSCENE_SCALE=128 -I/usr/include/n64/PR    -I/usr/include/n64 -O0    -MM src/util/dynamic_asset_loader.c build/assets/models/dynamic_animated_model_list.h build/assets/models/dynamic_model_list.h -MF "build/src/util/dynamic_asset_loader.d" -MT"build/src/util/dynamic_asset_loader.o"
mips-n64-gcc -c -mabi=32 -ffreestanding -mfix4300 -G 0     -DDEBUG -g -Isrc/ -I/usr/include/n64/nustd -Werror -Wall -DF3DEX_GBI_2 -DSCENE_SCALE=128 -I/usr/include/n64/PR    -I/usr/include/n64 -O0    -c -o build/src/util/dynamic_asset_loader.o src/util/dynamic_asset_loader.c
mips-n64-gcc -c -mabi=32 -ffreestanding -mfix4300 -G 0     -DDEBUG -g -Isrc/ -I/usr/include/n64/nustd -Werror -Wall -DF3DEX_GBI_2 -DSCENE_SCALE=128 -I/usr/include/n64/PR    -I/usr/include/n64 -O0    -MM src/util/dynamic_asset_data.c build/assets/models/dynamic_model_list_data.h -MF "build/src/util/dynamic_asset_data.d" -MT"build/src/util/dynamic_asset_data.o"
In file included from src/util/dynamic_asset_data.c:2:
src/../build/assets/models/dynamic_model_list_data.h:6:10: fatal error: cube/cube.h: No such file or directory
    6 | #include "cube/cube.h"`

--- a/Makefile
+++ b/Makefile
@@ -289,6 +289,7 @@ build/src/scene/security_camera.o: build/src/audio/clips.h build/assets/models/p
 build/src/scene/signage.o: $(MODEL_HEADERS)
 build/src/scene/switch.o: build/assets/models/props/switch001.h build/assets/materials/static.h build/assets/models/dynamic_animated_model_list.h
 build/src/util/dynamic_asset_data.o: build/assets/models/dynamic_model_list_data.h
+build/src/util/dynamic_asset_loader.o: build/assets/models/dynamic_model_list.h build/assets/models/dynamic_animated_model_list.h
 
 
 ANIM_TEST_CHAMBERS = build/assets/test_chambers/test_chamber_00/test_chamber_00_anim.o \
@@ -343,7 +344,7 @@ build/assets/test_chambers/level_list.h: $(TEST_CHAMBER_HEADERS) tools/generate_
 	@mkdir -p $(@D)
 	node tools/generate_level_list.js $@ $(TEST_CHAMBER_HEADERS)
 
-build/assets/models/dynamic_model_list.h build/assets/models/dynamic_model_list_data.h: $(DYNAMIC_MODEL_HEADERS) tools/generate_dynamic_model_list.js
+build/assets/models/dynamic_model_list.h build/assets/models/dynamic_model_list_data.h: $(DYNAMIC_MODEL_HEADERS) tools/generate_dynamic_model_list.js build/assets/models/cube/cube.h
 	@mkdir -p $(@D)
 	node tools/generate_dynamic_model_list.js build/assets/models/dynamic_model_list.h build/assets/models/dynamic_model_list_data.h $(DYNAMIC_MODEL_HEADERS)
 

